crates/bevy_pbr/src/render/mesh/mod.rs

```diff
@@ -656,6 +616,28 @@ impl MeshPipelineKey {
     }
 }
 
+fn has_skin(layout: &Hashed<InnerMeshVertexBufferLayout>) -> bool {
```

Comment:
```suggestion
fn is_skinned(layout: &Hashed<InnerMeshVertexBufferLayout>) -> bool {
```

Reply:
This is purely esthetic, but I find "has_skin" easier to read better than `is_skinned`, but given the convention here is to use "skinned" for skinned mesh, I'm happy to use `is_skinned`.